In 8.8 OpenSSH disabled sha1 rsa-sha keys out of the box,
so we need to re-enable them as a workaround for the test
suite until upstream updates the tests.

See: https://github.com/libssh2/libssh2/issues/630

--- tests/test_sshd.test
+++ tests/test_sshd.test
@@ -79,6 +79,8 @@ chmod go-rwx \
   -o "AuthorizedKeysFile ${PUBKEY} ${d}/openssh_server/authorized_keys" \
   -o "TrustedUserCAKeys ${cakeys}" \
   -o 'PermitRootLogin yes' \
+  -o 'HostKeyAlgorithms +ssh-rsa' \
+  -o 'PubkeyAcceptedAlgorithms +ssh-rsa' \
   -D \
   ${SSHD_FLAGS} &
 sshdpid=$!
